### DISCUSSION TITLE
GIMP is about to release an official AppImage: hurray!

### DISCUSSION THREAD

**Discussion by ivan-hc** [ivan-hc/GIMP-appimage#25] [Announcements]
My repository only served to overcome the deficiencies [in the database of "AM", my package manager](https://github.com/ivan-hc/AM), since an official AppImage did not exist... I also wanted to encourage developers to seriously consider adopting AppImage as a packaging format officially... and we are finally close to reaching this first goal!

https://www.gimp.org/news/2024/05/28/experiments-appimage

When the time comes, I will stop maintaining this repository. I don't think it makes sense to have to rename my package to "gimp-junest" for the stable version... how many would use it?

In any case, if something goes wrong in the future, keep my scripts in mind, they will certainly be useful to some of you.

I look forward to seeing many more official AppImages around. My goal here is this: to promote AppImage. And given your enormous support... I succeeded! Thank you all!

**Comment by in4matix** [comment]
Hi Ivan,
I know how you were instrumental in making this possible. I have been using your AppImage for about two years now and it works great without any issues. Thank you for commitment and hard work!
